common/graph_db_client.py

```diff
@@ -13,11 +13,12 @@
 import json
 
 class GraphDBClient:
-    def __init__(self,graph_db_params: GraphDBConfig|None):
+    def __init__(self,graph_db_params: GraphDBConfig|None,context_id: str|None):
+        self.username_prefix=graph_db_params.username
         self._client=client.Client(
             url=f"wss://{graph_db_params.account_name}.gremlin.cosmos.azure.com:443/",
             traversal_source="g",
-            username=graph_db_params.username,
+            username=self.username_prefix+"-contextid-"+context_id,
```

Comment:
why we have contextid in the username and we should incline it to support other types of auth.
in prod, infact, we won't be using username/password.

Reply:
This parameter is actually for the graph name, not really for authentication purposes. We can discuss more about what the naming format for the graphs should be